content/doc/book/pipeline/_chapter.yml

```diff
@@ -8,3 +8,5 @@ sections:
   - development
   - syntax
   - scaling-pipeline
+  - running-pipelines
```

Comment:
This should come earlier I think.  Just after getting started or after `jenkinsfile`.

Reply:
Will do.